src/amp-story-player/amp-story-player-impl.js

```diff
@@ -996,6 +996,29 @@ export class AmpStoryPlayer {
     });
   }
 
+  /**
+   * Updates the state of the features responsible for autoplay mode on the
+   * provided iframe.
+   * @param {number} iframeIdx
+   * @param {boolean} isAutoplaying
+   * @private
+   */
+  updateAutoplayState_(iframeIdx, isAutoplaying) {
```

Comment:
This would be called on your other PR's ` toggleAutoplayMode_()` method, right?

Reply:
Yes. As soon as that is merged I could make that change on this PR